lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -607,6 +615,7 @@ def __init__(self):
             ssh_password_enabled=dict(type='bool', default=True),
             ssh_public_keys=dict(type='list'),
             image=dict(type='dict'),
+            availability_set=dict(type='str'),
```

Comment:
add: default=None?

Reply:
Actually the default is None, so there is no need to add one

Reply:
Actually the default is None, so there is no need to add one